docs/examples/add-to-your-application/python/app/routes.py

```diff
@@ -0,0 +1,30 @@
+from flask import Flask
+from oso import NotFoundError
+from .models import users_db, Repository
+from .oso import oso
+
+
+app = Flask(__name__)
+
+def serialize(r):
```

Comment:
what do you think about making this `to_json` instead?

Reply:
could do that... this isn't exposed in the documentation so it doesn't really make a difference